src/app/utils/filterProtobufSchema.ts

```diff
@@ -0,0 +1,13 @@
+export const filterSchema = (schemas: string[]) => {
+  const invalidSchemas = [
```

Comment:
types, not schemas

schemas can contain multiple types.

filterInternalProtobufTypes